src/main/java/com/evolveum/polygon/connector/msgraphapi/UserProcessing.java

```diff
@@ -387,6 +401,14 @@ protected ObjectClassInfo objectClassInfo() {
         attrMail.setRequired(false).setType(String.class).setCreateable(true).setUpdateable(true).setReadable(true);
         userObjClassBuilder.addAttributeInfo(attrMail.build());
 
+        // UPDATE Extension ATTRIBUTES
+        for (int i = 1; i <= NUMBER_OF_EXTENSIONS; i++) {
```

Comment:
Please add the setReturnedByDefault(false) property for such an attribute, the attribute might not be needed in most deployments of the connector and additionally, it requires an additional roundtrip to the remote service. Also, please add the conditionals needed regarding the retrieval of the property which check "ATTRS_TO_GET" operation options in the correct places later in code.

Reply:
setReturnedByDefault(false), ATTRS_TO_GET in progress

Reply:
extension attributes are return by default (if you add them to selector) and they don't require additional round trip. So I don't see necessity of checking them against ATTR_TO_GET as the result would be another selector. So I personaly find this irrelevant. Can we agree on this? @matusmacik

Reply:
Hello, thank you. Yes in that case we can leave them in the schema.